github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -412,7 +408,6 @@ presubmits:
       preset-shared-images: "true"
     max_concurrency: 11
     name: pull-kubevirt-e2e-k8s-1.20-sig-compute-nonroot
-    optional: false
```

Comment:
This change makes two 1.20 nonroot jobs required.
But this is not done to others nonroot jobs in 1.20, and it does not appear at all for 1.21.

Reply:
The old value of `Optional: false` means that it was required already, please see: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#requiring-job-statuses

As the default value for Optional on a Presubmit is `false`, removing the `Optional: false` doesn't change anything regarding required status.

Reply:
Ahh it was false, right.